.checkignore

```diff
@@ -0,0 +1 @@
+test/*
```

Comment:
Please remove `.checkignore`. We don't need to exclude test files from static analysis.